packages/form/src/fields/ArrayField.tsx

```diff
@@ -0,0 +1,183 @@
+import React, { useContext } from 'react'
+
+import { useFieldArray, useFormContext } from 'react-hook-form'
+import { AttributeField } from './AttributeField'
+import { Button, Typography } from '@equinor/eds-core-react'
+import styled from 'styled-components'
+import { isPrimitive } from '../utils'
+import { useRegistryContext } from '../RegistryContext'
+// @ts-ignore
```

Comment:
ts ignore should not be necessary. probably something wrong with local setup or the package.json file for form plugin

Reply:
fixed